src/modules/EditProfile/components/UserCourseListItem/index.tsx

```diff
@@ -20,20 +22,33 @@ export const UserCourseListItem: FC<TUserCourseListItem> = ({
   onSub,
   course,
 }) => {
+  const userData = useSelector(selectUserData);
+
+  const { removeUserFromCourse } = useRemoveUserFromCourseMutation({
+    data: {
+      courseId: course.id,
+      userId: userData.id,
+      teamId:
+        userData.teams.find((team: Team) => team.courseId === course.id)?.id ??
+        null,
+      page: 0,
+    },
+  });
+  const onClickHandler = !deleteButton
```

Comment:
In which cases `deleteButton` can be false if we press `MinusButton`?

Reply:
It's because we have 2 lists of courses - user courses which can be deleted only throught the mutation and the local list of courses which can be changed locally without mutation and will be overwritten only after pressing the save button to update user info in edit profile

Reply:
Oh) Got it) Can we rename `deleteButton` to something more clear? deleteButton -> isLocal or localDelete for example?

Reply:
Agree with Yulia